doc/source/contribution/development/build/unix_cmake.rst

```diff
@@ -42,49 +42,15 @@ code from the repository::
 Run ``cmake``
 -------------
 
-You need to create ``Makefile`` for your environment.
-
-You can custom your build configuration by passing options to
-``cmake``.
-
-Here are recommended ``cmake`` options for developers::
-
-  % cmake . -DCMAKE_INSTALL_PREFIX=/tmp/local -DGRN_WITH_DEBUG=on -DGRN_WITH_MRUBY=on
-
-Here are descriptions of these options:
-
-``-DCMAKE_INSTALL_PREFIX=/tmp/local``
-
-    It specifies that you install your Groonga into temporary
-    directory. You can do "clean install" by removing
-    ``/tmp/local`` directory. It'll be useful for debugging install.
-
-``-DGRN_WITH_DEBUG=on``
-
-    It enables debug options for C/C++ compiler. It's useful for
-    debugging on debugger such as GDB and LLDB.
-
-``-DGRN_WITH_MRUBY=on``
-
-    It enables mruby support. The feature isn't enabled by default
-    but developers should enable the feature.
+See :ref:`cmake_run`.
```

Comment:
Could you use `-` not `_` as a word separator for reference ID?

```suggestion
See :ref:`cmake-run`.
```

Reply:
I have overlooked, fixed it.